### ISSUE TITLE
Splash screen accessibility

### ISSUE THREAD

**Issue by iamjolly** [GSA-TTS/10x-ai-sandbox#426]
# Description

While the splash screen is often shown very briefly (for a second or two), it could be shown for longer if someone is on a slow network or if there is more load on the server(s). 

The splash page is untitled, and the GSA logo is unlabelled (missing `alt` text) when it is shown. This causes screen readers to read out the filename, which is not very useful. 

## Proposed fixes

- Change the page `title` from Openweb UI to GSA Chat
- Add `alt` attribute(s) to the GSA logo(s) on the page. Contents should be GSA Chat.
